datadog/resource_datadog_synthetics_test_.go

```diff
@@ -197,6 +197,52 @@ func syntheticsTestOptions() *schema.Schema {
 					Type:     schema.TypeBool,
 					Optional: true,
 				},
+				"monitor_options": syntheticsTestMonitorOptions(),
+			},
+		},
+	}
+}
+
+func syntheticsTestMonitorOptions() *schema.Schema {
+	return &schema.Schema{
+		Type: schema.TypeMap,
+		DiffSuppressFunc: func(key, old, new string, d *schema.ResourceData) bool {
+			if key == "options.monitor_options.renotify_interval" {
+				// TF nested schemas is limited to string values only
+				// follow_redirects and accept_self_signed being booleans in Datadog json api
+				// we need a sane way to convert from boolean to string
+				// and from string to boolean
+				oldValue, err1 := strconv.ParseBool(old)
+				newValue, err2 := strconv.ParseBool(new)
```

Comment:
Should these two lines have `strconv.Atoi`? The `renotify_interval` field is integer, right?